public/apps/configuration/panels/audit-logging/audit-logging-edit-settings.tsx

```diff
@@ -155,11 +156,19 @@ export function AuditLoggingEditSettings(props: AuditLoggingEditSettingProps) {
   const renderComplianceSetting = () => {
     return (
       <>
-        <EuiPageHeader>
-          <EuiTitle size="l">
-            <h1>Compliance settings</h1>
-          </EuiTitle>
-        </EuiPageHeader>
+        <PageHeader
+          navigation={props.depsStart.navigation}
+          coreStart={props.coreStart}
+          fallBackComponent={
+            <EuiPageHeader>
+              <EuiTitle size="l">
+                <h1>Compliance settings</h1>
+              </EuiTitle>
+            </EuiPageHeader>
+          }
+          resourceType={ResourceType.auditLogging}
+          pageTitle="Compliance settings"
```

Comment:
Nit: Text constants should come from a constants file for fast translation processing.

Reply:
See above/created issue about doing this separately.